### PR TITLE
Added Verb to Download Images

### PR DESCRIPTION
**Pull request by auroq**
This adds the download verb with the same flags as run and test.
This also adds the download (with output) to the test.sh script.
I'm hoping having output during download will cause the build to pass even with c#

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -1,2 +1,11 @@
 #!/bin/bash
-exec python -m samplerunner test
+echo "#############################"
+echo "# DOWNLOADING DOCKER IMAGES #"
+echo "#############################"
+python -um samplerunner download
```
Comment (jrg94) [inline test.sh]: I think I want to get rid of this file altogether and rewrite the travis file to leverage the two commands in this file in different build phases.

**Comment by jrg94** [comment]
So, is it the image build process that's slow?

**Comment by auroq** [comment]
I've never used travis before, and I was misreading the output. It looks like the issue is not downloading the images. It's hanging on running one of the tests.

**Comment by auroq** [comment]
Looks like it is still C# that's holding up the build. I'm going to remove the testinfo.yml for that again. We can revist C# at a later time.

```
test/projects/test_baklava.py::test_baklava[baklava.py] PASSED           [  0%]
test/projects/test_baklava.py::test_baklava[baklava.go] PASSED           [  0%]
test/projects/test_baklava.py::test_baklava[Baklava.java] PASSED         [  0%]
test/projects/test_baklava.py::test_baklava[baklava.jl] PASSED           [  1%]
test/projects/test_baklava.py::test_baklava[Baklava.cs] 
```

File: test.sh (hunk 1 above)
Comment (auroq) [reply test.sh]: Agreed

**Comment by auroq** [comment]
@jrg94 This is now ready for merge

**Comment by jrg94** [comment]
Yeah, I was wondering if it was hanging because of a test or because it's taking so long to actually build the image. I imagine it doesn't hang locally. 

At any rate, merging!

**Pull request merged**